package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "vue-google-maps",
-  "version": "0.1.21",
+  "version": "1.0.0",
```

Comment:
This is cool, but the doesn't the actual name have to change, as it's no longer the same as the repo it was forked from?